### PR TITLE
Add hideHeader

### PR DESCRIPTION
**Pull request by ki38sato**
Add option flag to hide header.

### CODE REVIEW

**Comment by ReSTARTR** [comment]
@ki38sato How do you think rename the option form `hideHeader` to `hideheader` or `noheader`?

**Comment by ki38sato** [comment]
If anything, I like `noheader` or `no-header`

and `noheader` option in config might be unnecessary. What do you think?
(command-line is only enough?)

File: main.go [hunk 1]
```diff
@@ -63,6 +64,9 @@ func optionsFromFile() *client.Options {
 		opt.TagFilters = parseFilterString(cfg.Section("options").Key("tags").Value())
 		opt.Fields = parseFieldsString(cfg.Section("options").Key("fields").Value())
 		opt.Credentials = cfg.Section("options").Key("creds").Value()
+		if parsed, err := strconv.ParseBool(cfg.Section("options").Key("hideHeader").Value()); err == nil {
+			opt.HideHeader = parsed
+		}
```
Comment (ReSTARTR) [inline main.go]: It seems to be a little complecated... 
How about do this? 

```go
opt.HideHeader = (cfg.Section("options").Key("hideHeader").Value() == "true")
```

This allows only `true` chars. And, you don't need to use `strconv` package.

**Comment by ReSTARTR** [comment]
> `noheader` or `no-header`

OK, I'd like to choice `noheader`. :-)

> and noheader option in config might be unnecessary. What do you think?
> (command-line is only enough?)

Mmm... I think that it'd be convenient if I can define `noheader` in config.

**Comment by ki38sato** [comment]
changed to `noheader`, and remove `strconv` package.

**Comment by ReSTARTR** [comment]
Great:+1::+1::+1:
I'll merge this. 
Thank you :-)

**Pull request merged**